docs/Installation.md

```diff
@@ -585,60 +571,48 @@ sed -i '' '/[[:<:]]engine[[:>:]]/ s/=.*/= MySQL/' /usr/local/etc/zonemaster/back
 Install, configure and start database engine (and Perl bindings):
 
 ```sh
-pkg install mysql57-server p5-DBD-mysql
+pkg install -y mysql57-server p5-DBD-mysql
 sysrc mysql_enable="YES"
 service mysql-server start
 ```
 
-Read the current root password for MySQL:
+Read the current root password for MySQL (unless it has been changed
+already).
 
 ```sh
 cat /root/.mysql_secret
 ```
 
-Connect to MySQL interactively:
+Set password for MySQL root (required by MySQL). Use the password from 
+`/root/.mysql_secret` when prompted for password, and then the new password
+when prompted for that.
 
 ```sh
-mysql -u root -h localhost -p
-```
-
-Reset root password in MySQL (required by MySQL). Replace
-`<selected root password>` with the password from the file above
-(or another one of your choice):
-
-```sql
-ALTER USER 'root'@'localhost' IDENTIFIED BY '<selected root password>';
-```
-
-Unless you keep an old database, initialize the database:
-```sql
-CREATE DATABASE zonemaster;
-```
-
-Create a new user and give it all permissions on the newly created database:
-```sql
-CREATE USER 'zonemaster'@'localhost' IDENTIFIED BY 'zonemaster';
-GRANT ALL ON zonemaster.* TO 'zonemaster'@'localhost';
+/usr/local/bin/mysqladmin -u root -p password '<selected root password>'
 ```
 
-Logout from database:
+To create the database and the database user (unless you keep an old database).
+Edit the command first if you want a non-default database name, user name or
+password. Run the command on one line.  Use the MySQL root password when
+prompted.
 
-```sql
-exit;
+```sh
+mysql -u root -p -e "CREATE DATABASE zonemaster;" -e "CREATE USER 'zonemaster'@'localhost' IDENTIFIED BY 'zonemaster';" -e "GRANT ALL ON zonemaster.* TO 'zonemaster'@'localhost';"
 ```
 
-Then update the `/etc/zonemaster/backend_config.ini` file with username and
-password unless the default values are used.
+Update the `/etc/zonemaster/backend_config.ini` file with database name, username 

```

Comment:
Shouldn't it be `/usr/local/etc/zonemaster/backend_config.ini`?

Reply:
Yes. Good catch. Fixed.